dramatiq/results/backends/stub.py

```diff
@@ -29,7 +30,7 @@ class StubBackend(ResultBackend):
         result data.  Defaults to :class:`.JSONEncoder`.
     """
 
-    results = {}
+    results: Dict[str, Tuple[Optional[str], Optional[str]]] = {}
```

Comment:
the second value of the tuple is a float

Reply:
Thanks for spotting that. Fixed!